ocrd-olena-binarize

```diff
@@ -79,13 +76,43 @@ function image_id_from_fpath {
               -t -v "//mets:file[starts-with(@MIMETYPE,'image/') and ./mets:FLocat/@xlink:href='${image_in_fpath}']/@ID"
               "${ocrd__argv[mets_file]}" )
     if ! image_in_id=$(xmlstarlet "${options[@]}" | head -1); then
-        warning "image URL '${image_in_fpath}' not referenced"\
-                "in METS for input file ID=${in_id} (pageId=${in_pageId})"
+        ocrd log warning "image URL '${image_in_fpath}' not referenced in METS for input file ID=${in_id} (pageId=${in_pageId})" >&2
         image_in_id="${in_id}-IMG" # fallback
     fi
     echo "$image_in_id"
 }   
 
+function auto_winsize {
```

Comment:
clever.

Reply:
Hah! I have completely forgotten about this corner. This was neither anticipation of https://github.com/OCR-D/spec/pull/164 not recycling – I swear!